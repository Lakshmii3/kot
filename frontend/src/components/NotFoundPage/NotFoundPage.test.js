import React from 'react';
import { shallow } from 'enzyme';

import NotFoundPage from './NotFoundPage';

describe('Verify 404 default error page', () => {
    it('NoFoundPage should render correctly', () => {
        const errorPage = shallow(<NotFoundPage />);
        expect(errorPage).toMatchSnapshot();
    });
})