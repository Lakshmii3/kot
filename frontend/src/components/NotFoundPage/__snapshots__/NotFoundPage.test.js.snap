// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Verify 404 default error page NoFoundPage should render correctly 1`] = `
<Status
  code={404}
>
  <div
    className="NotFoundPage"
  >
    <h1>
      Oops!
    </h1>
    <div>
      Looks like you're lost...
    </div>
    <div
      className="action"
    >
      <a
        className="btn btn-primary"
        href="/"
      >
        Guide me to the right path!
      </a>
    </div>
  </div>
</Status>
`;
